### DISCUSSION TITLE
Add mechanism to auto-increment or decrement sliders periodically.

### DISCUSSION THREAD

**Discussion by lo5** [h2oai/wave#650] [Ideas]
Via @FelixArokia:

![animate_slider](https://user-images.githubusercontent.com/6334572/91329821-78b0c380-e77d-11ea-9585-e0d309602430.gif)

TBD: should this be a part of the existing slider, or a new component?

**Comment by mturoci** [comment]
This feature is not that complex as for implementation => will not pollute core slider code so I would say adding it to an existing component is sufficient.

wdyt @lo5 ?

**Comment by lo5** [comment]
Sounds good. I'm not too excited about the little play/pause button to the bottom right. Might look better if rendered to the right of the slider, vertically aligned with the slider's track. You might have better ideas.
